explanations/machine-learning-terms.md

```diff
@@ -25,6 +25,22 @@ Researchers often invent complicated names for simple things. This page will hel
     <p>One example is working with words in Natural Language Processing. Words are usually transformed to a list of numbers  (embedded) before given as input to a neural network. The exact mapping between word and the list of numbers will be automatically learned.</p>
 </td>
 </tr>
+<tr>
+<td valign="top">Principal Component Analysis (PCA)</td>
+<td>
+    <p>An algorithm that is used to narrow down datasets.</p>
+    <p>PCA (as any 'dimensionality reduction' algorithms) has several advantages:
+	<ul>
+		<li>it reduces the storage space required</li>
+		<li>it improves the performance of the machine learning model by removing the multicollinearity in the data</li>
+		<li>it eases the visualization of the data when reduced to very low dimensions such as 2D or 3D</li>
+	</ul>
+    </p>
+    <p>Technically, it's a statistical procedure that uses an orthogonal transformation to <strong>identify a small number of uncorrelated variables, called principal components</strong> from a large set of data possibly containing correlated variables.</p>
+    <p>This transformation is defined in such a way that the first principal component has the largest possible variance (variability in the data), and each succeeding component in turn has the highest variance possible under the constraint that it is orthogonal to the preceding components. So if we want to reduce from <tt>n</tt> to <tt>k</tt> dimensions, we keep the k's first principal components.</p>
```

Comment:
I really like the example! Maybe you can find a more intuitive way to express the variance/variability...

Reply:
"Technically, it's a statistical procedure that identifies and sorts a small number of uncorrelated variables, called principal components, in such a way that the first one has the most scattered data. So if we want to reduce from N to K dimensions, we keep the K's first principal components. But the lower is K the greater is the information loss."

This part is hard to summarize. Is the new approach easier to understand?

Reply:
Yes, this is much better!